rancher/rancher
# [v2.8] Bump wins `0.4.15-rc1`

## Issue: https://github.com/rancher/rancher/issues/45068
 
## Problem
When provisioning custom windows clusters several proxy environment variables will not be properly set for the rke2 service when provided as agent args in the Rancher UI. This can prevent proxies from working properly in certain environments. 
 
## Solution
A community user has raised a PR in rancher/wins which ensures that these proxy environment variables are properly attached to the rke2 windows service. rancher-wins v0.4.15-rc1 includes this fix, this PR simply bumps the wins version.
 
## Testing
## Engineering Testing
### Manual Testing


+ set up a squid proxy (or any kind of http proxy) 
+ Provision a custom RKE2 cluster with `Agent Env Vars` configured for `http_proxy`, `https_proxy`, and `no_proxy`
+ Add a windows node and ensure it becomes available 
+ Install [Sysinternals Process Explorer](https://learn.microsoft.com/en-us/sysinternals/downloads/process-explorer) in the windows worker node
+ Open Sysinternals Process Explorer and search rke2.exe and wins.exe
+ Right click on the process and choose Properties
+ Switch to tab Environment
+ Ensure that the proxy environment variables specified in the `Agent Env Vars` UI field are present

### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

## QA Testing Considerations

 
### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_
